isolate_example/README.md

```diff
@@ -9,6 +9,27 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate vs on a secondary isolate. 
```

Comment:
"Compares running a large computation on the main isolate with running the same calculation on a second isolate."